karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
Oops, checked docs https://github.com/douglasduteil/isparta

> Its intention is to be used with karma and karma-coverage, which provides code coverage reports using istanbul.

So far it is the only tool that gives a good es6 code coverage. I used isparta-loader, which just runs isparta instrumenter
```js
var instrumenter = new isparta.Instrumenter(assign(
        {
            embedSource: true,
            noAutoWrap: true
        },
        options
    ));
```

So it is possible to write some command-line utility to run isparta over the code and then run istanbul to get coverage report from it. But I am not sure it makes sense for the sake of not using karma. I would say getting pretty ES6 coverage is a good reason to use karma.

What do you reckon?